docs/reference/Signed-Packages-Reference.md

```diff
@@ -110,3 +110,76 @@ If a package has multiple owners, by default, "Any" owner's certificates can be
 Similarly, if the default "Any" option is selected for a package where one owner has a certificate registered and another owner does not have any certificate registered, then nuget.org accepts either a signed package with a signature registered by one of its owners or an unsigned package (because one of the owners does not have any certificate registered).
 
 ![Configure package signers](media/configure-package-signers.png)
+
+## Customize package consumption requirements
+
+*NuGet 4.9.0+ and Visual Studio version 15.9 and later*
+
+You can customize how NuGet clients validate package signatures by enabling the `signatureValidationMode` to `required`. 
```

Comment:
`enabling` -> `setting`? Also, `required` on line 118 doesn't match line 122:

```
    <add key="signatureValidationMode" value="require" />   
```

Should line 122 be updated?